src/java.xml/share/classes/module-info.java

```diff
@@ -842,6 +874,40 @@
  * <td style="text-align:center"><a href="#Processor">Method 1</a></td>
  * <td style="text-align:center">22</td>
  * </tr>
+ * <tr>
+ * <td id="JDKCATALOG_RESOLVE">{@systemProperty jdk.xml.jdkcatalog.resolve}</td>
+ * <td>Instructs the JDK default CatalogResolver to act in accordance with the setting
+ * of this property when unable to resolve an external reference with the built-in Catalog.
+ * The options are:
+ * <ul>
+ * <li><p>
+ * {@code continue} -- Indicates that the processing should continue
+ * </li>
+ * <li><p>
+ * {@code ignore} -- Indicates that the reference is skipped
+ * </li>
+ * <li><p>
+ * {@code strict} -- Indicates that the resolver should throw a CatalogException
+ * </li>
+ * </ul>
+ * </td>
+ * <td style="text-align:center">String</td>
+ * <td>
+ * {@code continue, ignore, and strict}. Values are case-insensitive.
+ * </td>
+ * <td style="text-align:center">continue</td>
+ * <td style="text-align:center">No</td>
+ * <td style="text-align:center">Yes</td>
+ * <td style="text-align:center">
+ *     <a href="#DOM">DOM</a><br>
+ *     <a href="#SAX">SAX</a><br>
+ *     <a href="#StAX">StAX</a><br>
+ *     <a href="#Validation">Validation</a><br>
+ *     <a href="#Transform">Transform</a>
+ * </td>
+ * <td style="text-align:center"><a href="#Processor">Method 1</a></td>
+ * <td style="text-align:center">22</td>
+ * </tr>
```

Comment:
I went through a number of iterations with Joe on this implNote so happy with this version.

Reply:
Thanks Alan! I really appreciate your allocating so much time to this project, esp. the APIs and CSRs. So this is the last piece to finally complete the project.